packages/@expo/cli/src/start/server/AsyncNgrok.ts

```diff
@@ -51,7 +51,8 @@ export class AsyncNgrok {
     return [
       // NOTE: https://github.com/expo/expo/pull/16556#discussion_r822944286
       await this.getProjectRandomnessAsync(),
-      slugify(username),
+      // Strip out periods from the username to avoid subdomain issues with SSL certificates.
+      slugify(username, { remove: /\./ }),
```

Comment:
I see that the `slugify` docs say this:
> * If the value of `remove` is a regular expression, it should be a [character class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes) and only a character class. It should also use the [global flag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/global). (For example: /[*+~.()'"!:@]/g.) Otherwise, the remove option might not work as expected.
> * If the value of `remove` is a string, it should be a single character. Otherwise, the remove option might not work as expected.

So it seems they're saying you should use `'.'` or `/[.]/g`

However, I tested `/\./` and it seems to work fine! I suppose their recommendation is just a [lie to children](https://en.wikipedia.org/wiki/Lie-to-children)?